### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by Serco627**
first pull request test

### CODE REVIEW

**Comment by Serco627** [comment]
maybe you add some more text

**Comment by anapecic** [comment]
test kommentar

File: git-cheatsheet.md [hunk 1]
```diff
@@ -14,3 +14,15 @@ git switch <branchname> switch branches
 git branch list your branches
 git branch -a list all branches (local and remote)
 git branch -d <branchname> delete a branch
+
+## Basic Workflow for a pull request
+
+Create a new branch with git switch -c <branchname>.
+Make changes to the code / write your code fpr the feature.
+Push the changes and the new branch with git push -u origin <branchname> (after you have done this once you can use git push for this branch)
```
Comment (anapecic) [inline git-cheatsheet.md:22]: test review

**Review by anapecic — CHANGES_REQUESTED** [review]
commented in l22

**Review by JohannMerck — COMMENTED** [review]
1a

**Pull request merged**